Apps/Common/src/Services/PatientService.cs

```diff
@@ -94,58 +98,28 @@ public async Task<RequestResult<string>> GetPatientPhn(string hdid)
         public async Task<RequestResult<PatientModel>> GetPatient(string identifier, PatientIdentifierType identifierType = PatientIdentifierType.Hdid, bool disableIdValidation = false)
         {
             using Activity? activity = Source.StartActivity();
-            RequestResult<PatientModel> requestResult = new();
             PatientModel? patient = this.GetFromCache(identifier, identifierType);
-            if (patient == null)
```

Comment:
There is also Patient.Services.PatientService.  That is currently for V2.  What is changed here is for V1.  Does this change need to be done for V2?

Reply:
I did not work on V2 yet as I didn't refactor any referencing top level services yet. The change here is for readability - inverting ifs to make the happy path flows through the procedure.

Reply:
I did not work on V2 yet as I didn't refactor any referencing top level services yet. The change here is for readability - inverting ifs to make the happy path flows through the procedure.